docs-app/app/assets/styles.css

```diff
@@ -5,6 +5,6 @@
 /**
  * postcss-import doesn't follow "exports" entries in package.json
  */
-@import '@crowdstrike/ember-oss-docs/dist/styles/tailwind-extensions.css';
-@import '@crowdstrike/ember-oss-docs/dist/styles/docfy-overrides.css';
-@import '@crowdstrike/ember-oss-docs/dist/styles/hljs.css';
+@import '@crowdstrike/ember-oss-docs/styles/tailwind-extensions.css';
+@import '@crowdstrike/ember-oss-docs/styles/docfy-overrides.css';
+@import '@crowdstrike/ember-oss-docs/styles/hljs.css';
```

Comment:
@NullVoxPopuli the problem with this now is that if you visit https://e164cf93.ember-toucan-core.pages.dev/, you'll see the CSS not being applied.  Do we need to tell Webpack about this by adding a CSS Loader of sorts?

Reply:
It's possible that https://github.com/CrowdStrike/ember-oss-docs/blob/main/ember-oss-docs/package.json#L167 needs to be expanded to differentiate between js and css. 
otherwise, it wouldn't be too farfetched to include `./dist/styles` in exports so that both webpack and the css-loader are happy.

it turns out googling `exports` and `webpack` together is really hard, due to CJS :sweat_smile: 

Some back and forth with AI suggested using `resolve.alias`, but that's less fun. may work tho

Reply:
oh, alternatively, these 3 files could be imported in a js file.